securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -317,7 +317,10 @@ Redirects to login form on (options is an optional object with redirectUri and/o
 Options is an Object, where:
 
 * redirectUri - Specifies the uri to redirect to after login.
-* prompt - By default the login screen is displayed if the user is not logged-in to {project_name}. To only authenticate to the application if the user is already logged-in and not display the login page if the user is not logged-in, set this option to `none`. To always require re-authentication and ignore SSO, set this option to `login` .
+* prompt - By default the login screen is displayed if the user is not logged-in to {project_name}. To only authenticate to the
+application if the user is already logged-in and not display the login page if the user is not logged-in, set this option to `none`.
+To always require re-authentication and ignore SSO, set this option to `login` .
+To always show the consent page even if user already granted consent to this client application, set this option to `consent` (Applicable only for the clients with `Consent Required`) .
```

Comment:
Suggested edits:

* prompt - By default, the login screen is displayed if the user is not logged-in to {project_name}.
Set this option to `none` to authenticate to the application if the user is already logged-in and to not display the login page if the user is not logged-in.
Set this option to `login` to always require re-authentication and ignore current logged-in status and SSO.
Set this option to `consent` to always show the consent page, even if user already granted consent to this client application (this is applicable only for the clients with `Consent Required` set).